DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/ViewDelegate/_DelegateCourseInfoCard.cshtml

```diff
@@ -150,13 +159,24 @@
         View progress
       </a>
       <a class="nhsuk-button nhsuk-button--secondary" role="button">See learning log</a>
-      <a
-        class="nhsuk-button delete-button"
-        role="button"
-        asp-controller="ViewDelegate"
-        asp-action="ConfirmRemoveFromCourse"
-        asp-route-delegateId="@Model.DelegateId"
-        asp-route-customisationId="@Model.CustomisationId">
+
+      @if (Model.IsProgressLocked) {

```

Comment:
Please could you note down this additional button in the Jira ticket requirements so that if can be checked during testing.

Reply:
Updated jira thanks